modules/react/collection/stories/Collection.stories.mdx

```diff
@@ -155,38 +169,15 @@ columns, but not rows. This is the default navigation manager for lists.
 
 ## Hooks
 
-### `useListItemRegister`
-
+<>
 <SymbolDoc name="useListItemRegister" fileName="/react/" />
-
-### `useListItemAllowChildStrings`
-
 <SymbolDoc name="useListItemAllowChildStrings" fileName="/react/" />
-
-### `useListItemRovingFocus`
-
 <SymbolDoc name="useListItemRovingFocus" fileName="/react/" />
-
-### `useListItemSelect`
-
 <SymbolDoc name="useListItemSelect" fileName="/react/" />
-
-### `useListRenderItems`
-
 <SymbolDoc name="useListRenderItems" fileName="/react/" />
-
-### `useListResetCursorOnBlur`
-
 <SymbolDoc name="useListResetCursorOnBlur" fileName="/react/" />
-
-### `useOverflowListItemMeasure`
-
 <SymbolDoc name="useOverflowListItemMeasure" fileName="/react/" />
-
-### `useOverflowListMeasure`
-
 <SymbolDoc name="useOverflowListMeasure" fileName="/react/" />
-
-### `useOverflowListTarget`
-
 <SymbolDoc name="useOverflowListTarget" fileName="/react/" />
```

Comment:
I didn't catch this before, but the headings were listed twice. I think it is cleaner to use the headings included in the `SymbolDoc>` rather than pass `hideHeading` to `SymbolDoc` and duplicate the heading strings.